autoload/ddu/ui/ff.vim

```diff
@@ -457,21 +432,64 @@ function ddu#ui#ff#_stop_debounce_timer(timer_name) abort
   endif
 endfunction
 
-function s:do_cursor_moved(winid) abort
-  const prev_winid = win_getid()
-  try
-    call win_gotoid(a:winid)
+function ddu#ui#ff#_open_filter_window(input, params) abort
+  let s:filter_prev_input = a:input
+  let s:filter_updatetime = a:params.filterUpdateTime
 
-    silent doautocmd CursorMoved
-  finally
-    call win_gotoid(prev_winid)
-  endtry
+  augroup ddu-ff-filter
+    autocmd!
+    autocmd CmdlineChanged <buffer> call s:check_update()
+    autocmd User Ddu:ui:ff:openFilterWindow :
+    autocmd User Ddu:ui:ff:closeFilterWindow :
+  augroup END
+
+  doautocmd User Ddu:ui:ff:openFilterWindow
+
+  const input = exists('*cmdline#input') ?
+        \ cmdline#input(a:params.prompt, a:input) :
+        \ input(a:params.prompt, a:input)
+
+  augroup ddu-ff-filter
+    autocmd!
+  augroup END
+  let s:filter_prev_input = input
+  doautocmd User Ddu:ui:ff:closeFilterWindow
```

Comment:
Display warning when using Vim, because `Ddu:ui:ff:closeFilterWindow` fires after `ddu-ff-filter` group is disposed.

Reply:
Fixed.